saleor/order/emails.py

```diff
@@ -43,6 +45,38 @@ def collect_data_for_email(order_pk, template):
     }
 
 
+def collect_staff_data_for_email(order_pk, template):
+    """Collect the required data for sending emails.
+
+    Args:
+        order_pk (int): order primary key
+        template (str): email template path
+
+    """
+    order = Order.objects.get(pk=order_pk)
+    staff_notifications = StaffNotification.objects.filter(
+        active=True, user__is_active=True
+    )
+    recipient_emails = [notification.email for notification in staff_notifications]
```

Comment:
Maybe `recipient_emails  = StaffNotification.objects.filter(...).values_list("email", flat=True)` ?

Reply:
It can't be like this because `email` is a method right now. Field for the email is `staff_email`.

Reply:
`email` is property
`staff_email` id field that can be null if user is set
`email` takes `staff_email` or `user.email` depends on which field is set

Reply:
Oh true, I missed decorator.